generators/uscore-r4/templates/sequence.rb.erb

```diff
@@ -26,7 +26,8 @@ module Inferno
       test '<%=test[:tests_that]%>' do
         metadata do
           id '<%=format('%02d', test[:index])%>'
-          link '<%=test[:link]%>'
+          link '<%=test[:link]%>'<%='
+          optional' if test[:optional]%>
```

Comment:
Having the erb tag split over multiple lines looks really weird to me. I'd definitely prefer to have the whole tag on one line, or to do `<% if test[:optional] %>optional<% end %>`.

Reply:
Couldn't make it one line due to newline issues. Used the ` if test[:optional]` approach.